docs/monitoring/metrics.md

```diff
@@ -94,8 +94,8 @@ Alternatively you can also use your own `Counter` implementation:
 <div data-lang="java" markdown="1">
 {% highlight java %}
 
-public class MyMapper extends RichMapFunction<String, String> {
-  private Counter counter;
+RichMapFunction<String, String> {
```

Comment:
looks like you removed more than intended?

Reply:
Now this PR has been updated, please look at it again.